### PR TITLE
Add support for multiple authors

### PR DESCRIPTION
**Pull request by dieterve**
### Changed

* Allow multiple authors to be defined
* Abstract the author view so its the same everywhere

-- 

Example:
screenshot 2016-07-01 15 02 25

### CODE REVIEW

File: _includes/partials/author.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% if post.authors %}
+    {% assign authors = post.authors %}
+{% elsif page.authors %}
+    {% assign authors = page.authors %}
+{% else %}
+    {% assign authors = null %}
+{% endif %}
```
Comment (hannesvdvreken) [inline _includes/partials/author.html]: you could also do a check here:

```
{% if authors.is_a?(String) %}
    {% assign authors = [authors] %}
{% endif %}
```

This way you don't need to change all old article files.

Comment (dieterve) [inline _includes/partials/author.html]: I would still need to change all old files because of `author => authors`

Comment (hannesvdvreken) [inline _includes/partials/author.html]: ah yes

**Comment by dieterve** [comment]
🚢 ?

**Comment by jdrieghe** [comment]
Looks good to me!

**Comment by dieterve** [comment]
Anyone else? Merging in 15min

**Comment by dieterve** [comment]
Did some changes, for me its oke now. @bramdevries can you also check again?

@yannickdepauw: any ideas for an image?

**Comment by hannesvdvreken** [comment]
:ship:

**Pull request merged**